db.go

```diff
@@ -53,3 +53,32 @@ func (m *mongoDepot) DeleteExpiresBefore(cutoff time.Time) error {
 func expiresBeforeQuery(cutoff time.Time) bson.M {
 	return bson.M{userTTLKey: bson.M{"$lte": cutoff}}
 }
+
+// FindUser finds the User in the database by name.
+func (m *mongoDepot) FindUser(name string) (*User, error) {
```

Comment:
do we need to change the interface to make these more broadly accessible?

Reply:
No, see this [ticket](https://jira.mongodb.org/browse/MAKE-822)